web/profiles/joinup/js/site_wide_featured.js

```diff
@@ -6,6 +6,16 @@
 (function ($, Drupal) {
 
   /**
+   * Theme function that renders a visual cue for featured content.
+   *
+   * @returns {string}
+   *   The HTML for the featured content visual cue.
+   */
+  Drupal.theme.featuredContentCue = function () {
+    return '<div class="listing__corner listing__corner--featured"><span class="icon icon--star"></span></div>';
```

Comment:
`listing__corner--featured` seems be unnecessary - I couldn't find any reference usage for it, similarly to the `listing__corner--pin`.

Reply:
Done.